site/src/components/segmented-controls.md

```diff
@@ -0,0 +1,401 @@
+			<p>Tab Item 6</p>
+		</div>
+	</div>
+</tab-container>
+```
+
+## Segmented Control with footer Next/Prev
+
+Removing the next and previous buttons will still provide a seamless experience on the tabs control. Small restructuring allows for the tabs-container component to allocate the `after-tabpanel` footer.
+
+```html
+<tab-container data-segmented-controls-container>
+	<div class="segmented-controls" role="tablist" aria-orientation="horizontal">
+		<div class="segmented-control-list">
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-three-1"
+				role="tab"
+				aria-selected="true"
+				data-segmented-control
+			>
+				Define Strategy
+			</button>
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-three-2"
+				role="tab"
+				data-segmented-control
+			>
+				Plan
+			</button>
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-three-3"
+				role="tab"
+				data-segmented-control
+			>
+				Prepare
+			</button>
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-three-4"
+				role="tab"
+				data-segmented-control
+			>
+				Adopt
+			</button>
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-three-5"
+				role="tab"
+				data-segmented-control
+			>
+				Govern
+			</button>
+			<button
+				class="segmented-control"
+				type="button"
+				id="sc-three-6"
+				role="tab"
+				data-segmented-control
+			>
+				Manage
+			</button>
+		</div>
+	</div>
+
+	<div role="tabpanel" aria-labelledby="sc-three-1">
+		<div class="margin-block-md text-align-center">
+			<p>Tab Item 1</p>
+		</div>
+	</div>
+	<div role="tabpanel" aria-labelledby="sc-three-2" hidden>
+		<div class="margin-block-md text-align-center">
+			<p>Tab Item 2</p>
+		</div>
+	</div>
+	<div role="tabpanel" aria-labelledby="sc-three-3" hidden>
+		<div class="margin-block-md text-align-center">
+			<p>Tab Item 3</p>
+		</div>
+	</div>
+	<div role="tabpanel" aria-labelledby="sc-three-4" hidden>
+		<div class="margin-block-md text-align-center">
+			<p>Tab Item 4</p>
+		</div>
+	</div>
+	<div role="tabpanel" aria-labelledby="sc-three-5" hidden>
+		<div class="margin-block-md text-align-center">
+			<p>Tab Item 5</p>
+		</div>
+	</div>
+	<div role="tabpanel" aria-labelledby="sc-three-6" hidden>
+		<div class="margin-block-md text-align-center">
+			<p>Tab Item 6</p>
+		</div>
+	</div>
+
+	<!-- After Tabpanels -->
+	<div class="border-top display-flex padding-top-sm">
+		<button
+			type="button"
+			class="button button-sm button-primary margin-right-sm"
+			title="Previous segment"
+			data-segmented-control-nav="previous"
+		>
+			<span class="icon" aria-hidden="true">
+				<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 448">
+					<path
+						class="fill-current-color"
+						d="M448 224H62.625l148.688 148.688-22.625 22.625L1.375 208 188.688 20.688l22.625 22.625L62.625 192H448v32z"
+					/>
+				</svg>
+			</span>
+			Previous
+		</button>
+		<button
```

Comment:
I can't make out a focus outline on this "Next" button. I also can't find a clear reason why that is, even on a cursory look through DevTools.

Reply:
On further investigation, the outline *does* appear, it's just showing as black on the dark mode's dark background. 

I think this is a case of our `outline-color: inherit` failing to break through the shadow DOM barrier.

Reply:
Right and because the buttons are outside of that `segmented-controls` class it wont catch the outline.  Might need an atomic here.

Reply:
LGTM